components/AppShell.tsx

```diff
@@ -27,7 +28,7 @@ const AppShell = () => {
       <IonReactRouter>
         <IonRouterOutlet id="main">
           <Route path="/tabs" render={() => <Tabs />} />
-          <Route path="/" render={() => <Redirect to="/tabs/feed" />} exact={true} />
+          <Route path="/tabs/feed" render={() => <Redirect to="/tabs/feed" />} exact={true} />
         </IonRouterOutlet>
```

Comment:
This looks like it removes the redirect from / to /tabs/feed, did this mean to be reverted?

Reply:
I was receiving an error from ionicons `Unabled to get property 'isProxied' of undefined or null reference`, a somewhat obscure error since it didn't stack trace, it only provided a URL to ionicons. During research I came across [a solution for a similar issue](https://github.com/ionic-team/stencil/issues/1661#issuecomment-719019126). Tinkering a bit, I changed the only path in the app that was directing to "/", which was this line, and it immediately stopped the error. 

Going back, however, I've changed that line back to its former "/", and there's now no error. It may have been an error that was fixed by a VS Code reload (there was another such during my refactor). I'd say it can be changed back, but if that error pops up again, that's the culprit. 

Also, a type error has just appeared in the same file, line 28, `<IonReactRouter>: Type '{ children: Element; }' has no properties in common with type 'IntrinsicAttributes & IntrinsicClassAttributes<IonReactRouter> & Readonly<IonReactRouterProps>'.` I'll fix that and update the PR, or start a new PR if this one gets approved beforehand.

Reply:
Update, I've made some updates but noticed more errors, so I'm going to circle back soon :-)

I've included another Route path in the next commit, as it seems to silence the again-persistent isProxied error I'd mentioned above. Let's see if it does the trick!

I also ignored a TS error in `/components/pages/Tabs.tsx` which offered an error for a seeming non-issue.

And finally, I removed as many unused import statements as I could come across.